.eslintrc.json

```diff
@@ -9,18 +9,11 @@
         "sinon": true
     },
     "extends": ["airbnb", "prettier"],
-    "plugins": ["chai-friendly", "jsdoc", "prettier", "unicorn"],
+    "plugins": ["chai-friendly", "prettier", "unicorn"],
     "parserOptions": {
         "sourceType": "module",
         "ecmaVersion": 2020
     },
-    "settings": {
-        "jsdoc": {
-            "tagNamePreference": {
-                "returns": "return"
-            }
-        }
-    },
```

Comment:
I'm *perfectly fine* with removing the `settings` portion, and if it turns out we're not depending on the plugin removing that too. But to be clear, this is not part of the doc generation.

Reply:
Yeah, I guess we might as well keep that consistent. Will undo.